### ISSUE TITLE
[Keyboard Navigation - Azure Search Services - Search Explorer]: The arrow buttons present under "results" pane are not accessible by keyboard.

### ISSUE THREAD

**Issue by rishuSrivastava11hcl** [microsoft/monaco-editor#2267]
### **User Experience:** 
If the controls or buttons present on the screen are not accessible by keyboard then it will be an issue for keyboard users to perform the functionality related to the button and will also miss important information regarding that button or control.

### **Repro Steps:**

1. Launch the URL: https://ms.portal.azure.com/#allservices.
2. Login with valid credentials
3.  Navigate to All Services and search "Search Services."
4. Select the already created service.
5. Select "Search Explorer" button. New blade starts appearing.
6. Turn on Tabstops and navigate to "Results" pane.
7. Verify the issue.

### **Actual Result:**

The arrow buttons present under "Results" pane are not accessible by keyboard. The keyboard focus moves out of the results pane and move to the title of the page and does not go to arrow buttons present on the left side.

### **Expected Result:**

The arrow buttons present under "Results" pane should be accessible by the keyboard. The keyboard focus should not move out of the results pane but should the arrow buttons present on the left side So that the keyboard user can expand collapse the query results.

### **Note:**
Same issue exists in similar panes throughout the application.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at https://aka.ms/AccessibilityInsights.

### **WCAG Reference Link:**
https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation.html

![MAS2 1 1_Arrow buttons under results pane are not accessible by keyboard](https://user-images.githubusercontent.com/72869551/102210100-c2bde080-3ef7-11eb-87fa-371530e6995a.png)

**Comment by rishuSrivastava11hcl** [comment]
#Accessibility;#WCAG2.1.1;#HCL;#HCL-AzureSearch-BM;#A11ySev1;#Benchmark;#Win10-Edge(Chromium);#A11yAuto;#Tabstop;#k4W

**Comment by alexdima** [comment]
From the screenshot, I can guess that this is about the folding indicators. Let's track in https://github.com/microsoft/vscode/issues/43262
